openapi/components/schemas/SubscriptionOrder.yaml

```
@@ -333,6 +333,16 @@ properties:
       If this value is not passed during order creation,
       a [pending order TTL](https://all-rebilly.redoc.ly/tag/Organizations/operation/PatchOrganization/#!path=settings/billing/pendingOrderTtl&t=request) setting is used to calculate the value.
     format: date-time
+  delinquencyPeriod:
+    type:
+      - 'string'
+      - 'null'
+    description: |-
+      Length of time, in ISO-8601 durations format,
+      before which an order with an invoice in past due state is automatically canceled.
+      If provided value is `null`, order does not change a state.
```

suggestion: for readability
```suggestion
      Length of time, in ISO-8601 durations format,
      before which an order with an invoice in past the due state is automatically canceled.
      If the provided value is `null`, the order does not change a state.
```